.github/workflows/release.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: npm install
+      - run: npm version "$(basename ${{ github.ref }})"
```

Comment:
To allow this the release should be done using tags triggered by tags event

Reply:
also maybe better to use

```suggestion
      - run: npm version "$(git describe --tags)"
```

to support `on.workflow_dispatch`?

Reply:
Not neccessarily: [`release`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release) does also work and integrates a bit better with GitHub.

Reply:
Using `on.workflow_dispatch` was just a matter of convenience for testing, but should probably be removed if we plan to automate this part of the release process.